### PR TITLE
Added navbar

### PR DESCRIPTION
**Pull request by Anton-van-Niekerk**
Added and styled a basic navbar.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>NIC | National Irrigation Consultants</title>
```
Comment (madeleinvanniekerk) [inline index.html]: please attach a screenshot in the PR description

Comment (Anton-van-Niekerk) [reply index.html]: Screenshot_2020-02-28 NIC National Irrigation Consultants

File: index.html [hunk 2]
```diff
@@ -7,6 +7,22 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
+<<<<<<< HEAD
```
Comment (madeleinvanniekerk) [inline index.html]: need to delete this line

File: index.html [hunk 3]
```diff
@@ -7,6 +7,22 @@
...
 </body>
-</html>
\ No newline at end of file
+</html>
+
+=======
+    <script src="main.js"></script>
+</body>
+</html>
+>>>>>>> master
```
Comment (madeleinvanniekerk) [inline index.html]: and these lines. It's part of the merge conflict

File: style.css [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    height: 1px;
+    background: #fff;
+    transition: width 0.6s ease 0s, left 0.6s ease 0s;
+    width: 0;
+}
+
+ul li a:hover:after {
+    width: 100%;
+    
+}
```
Comment (madeleinvanniekerk) [inline style.css]: as a rule we always add a newline at the end of a file. The lack of newline is indicated by the red icon

Comment (Anton-van-Niekerk) [reply style.css]: Noted, tx.

> as a rule we always add a newline at the end of a file. The lack of newline is indicated by the red icon

noted, tx

**Pull request merged**